rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
Why do you consider `PUT` as a safe method? Should not be only who has `edit taxonomy` permission to be able to perform a `PUT` request?
cc @pdpinch